### PR TITLE
Fix/refactor thomas

### PR DESCRIPTION
**Pull request by thawley2**
Separated game class game play methods to a new class called GamePlay, converted all methods to work with the split.  
Also created new player_turn and computer_turn methods to re-ask for input if an invalid selection was chosen.

### CODE REVIEW

File: lib/game_play.rb [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+      puts `clear`
+      puts "#{ans} isn't a valid answer fool!!"
+      game_menu
+    end
+  end
+
+  def turn_round
+    until @game.game_win != nil || @game.game_draw == true
+      puts `clear`
+      puts "You chose #{@player_turns.last} and the computer chose #{@comp_turns.last}\n\n==============="
```
Comment (reidsmiller) [inline lib/game_play.rb]: Need to add `if @player_turns != []`

File: lib/game_play.rb [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    elsif @game.game_win == false
+      puts `clear`
+      @comp_wins += 1
+      puts "YOU LOST! You're not smart enough to beat a computer choosing random columns? Sad day."
+    elsif @game.game_draw == true
+      puts `clear`
+      @draws += 1
+      puts "That was a draw. BOOOOOOOORRRRRRIIIIIINGGGGGGGG. You're literally not smart enough to beat a computer choosing random columns."
+    end
+    puts "================================\n\nYou've won #{@player_wins} times\n\nThe computer has won #{@comp_wins} times\n\nYou've had #{@draws} draw matches\n\n================================\n\nPress any key to continue"
```
Comment (reidsmiller) [inline lib/game_play.rb]: Change last bit to 'Press Enter to continue' instead of any key

**Review by reidsmiller — CHANGES_REQUESTED** [review]
Two changes requested, see specific comments

File: lib/game_play.rb (hunk 1 above)
Comment (thawley2) [reply lib/game_play.rb]: Good call, will re-submit in a few.

File: lib/game_play.rb (hunk 2 above)
Comment (thawley2) [reply lib/game_play.rb]: Will do!!

**Review by reidsmiller — APPROVED** [review]
Looks good!

**Pull request merged**